i8_terminal/commands/metrics/metrics_historical.py

```diff
@@ -332,18 +336,25 @@ def historical(
     with console.status("Fetching data...", spinner="material") as status:
         df = get_historical_metrics_df(tickers_list, metrics_list, period_type, from_date, to_date)
         df = df.sort_values(["PeriodDateTime"], ascending=False).groupby(["Ticker", "Metric", "Period"]).head(1)
-        if len(df["default_period_type"].unique()) > 1:
+        if not period_type:
+            if len(df["default_period_type"].unique()) > 1:
+                console.print(
+                    "The `period type` of the provided metrics are not compatible. Make sure the provided metrics have the same period type. Check `metrics describe` command to find more about metrics.",  # noqa: E501
+                    style="yellow",
+                )
+                return
+        if len(df["metric_name"].unique()) < len(metrics_list):
```

Comment:
How come `len(df["metric_name"].unique())` can be smaller than `len(metrics_list)`? Isn't metrics_list the same as the metrics in df? Otherwise, write a comment or use better variable names to make the code more readable.